lib/phoenix/live_dashboard/helpers.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Phoenix.LiveDashboard.Helpers do
   @moduledoc false
 
+  alias Phoenix.LiveDashboard.PageBuilder
```

Comment:
i don't like we have a cyclic dependency now between page builder and helpers :( What if we move live_dashboard_path to page builder and then @doc false it?

Reply:
@josevalim I could remove the cyclic dependency by doing what you suggested! o/
But some of the `format_*` functions from the Helpers module still uses the `encode_*` functions. I think this is OK. WDYT?

Reply:
It is ok, yes!